fed/mosaic-application/src/main/java/org/eclipse/mosaic/fed/application/ambassador/simulation/perception/SimplePerceptionConfiguration.java

```diff
@@ -54,4 +53,25 @@ public double getViewingRange() {
     public List<PerceptionModifier> getPerceptionModifiers() {
         return perceptionModifiers;
     }
+
+    public static class Builder {
+        private final double viewingAngle;
+        private final double viewingRange;
+
+        private final List<PerceptionModifier> perceptionModifiers = new ArrayList<>();
+
+        public Builder(double viewingAngle, double viewingRange) {
+            this.viewingAngle = viewingAngle;
+            this.viewingRange = viewingRange;
+        }
+
+        public Builder addModifier(PerceptionModifier perceptionModifier) {
+            perceptionModifiers.add(perceptionModifier);
+            return this;
+        }
+
+        public SimplePerceptionConfiguration build() {
+            return new SimplePerceptionConfiguration(viewingAngle, viewingRange, perceptionModifiers);
```

Comment:
put perceptionModifiers into a new list. otherwise anything what would be added to the builder after calling "build" would still be applied to the configuration object which has already been build.